internal/util/importutilv2/reader.go

```diff
@@ -0,0 +1,49 @@
+// Licensed to the LF AI & Data foundation under one
+// or more contributor license agreements. See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership. The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package importutilv2
+
+import (
+	"github.com/milvus-io/milvus-proto/go-api/v2/schemapb"
+	"github.com/milvus-io/milvus/internal/storage"
+	"github.com/milvus-io/milvus/internal/util/importutilv2/binlog"
+)
+
+type Reader interface {
+	Next(count int64) (*storage.InsertData, error)
+	Close()
+}
+
+type ColumnReader interface {
+	Next(count int64) (storage.FieldData, error)
```

Comment:
The term "Next(count)" has inconsistent semantics, especially when it comes to JSON and binlog readers, as it carries different meanings. I suggest renaming it and removing the "count" parameter. Instead, consider using a parameter that reflects the specific configuration of the reader object during its construction.